apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/hooks/useCaseBlocksLogic/useCaseBlocksLogic.ts

```diff
@@ -0,0 +1,45 @@
+import { TWorkflowById } from '@/domains/workflows/fetchers';
+import { useCaseTabsLogic } from '@/lib/blocks/variants/DefaultBlocks/hooks/useCaseBlocksLogic/hooks/useCaseTabsLogic/useCaseTabsLogic';
+import { getTabsToBlocksMap } from '@/lib/blocks/variants/DefaultBlocks/hooks/useCaseBlocksLogic/utils/getTabsToBlocksMap';
+import { getVariantBlocks } from '@/lib/blocks/variants/DefaultBlocks/hooks/useCaseBlocksLogic/utils/getVariantBlocks';
+import { getVariantTabs } from '@/lib/blocks/variants/DefaultBlocks/hooks/useCaseBlocksLogic/utils/getVariantTabs';
+import { useMemo } from 'react';
+
+export type TCaseBlocksCreationProps = {
+  workflow: TWorkflowById;
+  onReuploadNeeded: any;
+  isLoadingReuploadNeeded: boolean;
+};
+
+export const useCaseBlocksLogic = (
+  allBlocks: any[],
```

Comment:
Is the `Blocks` type from `@ballerine/blocks` the one you were looking for? Or a return type of a different function?
```suggestion
  allBlocks: Blocks,
```